src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen.py

```diff
@@ -324,7 +295,7 @@ def __call__(
         do_classifier_free_guidance = guidance_scale > 1.0
 
         # 1. Check inputs. Raise error if not correct
-        image_embeddings, prompt, negative_prompt, num_inference_steps = self.check_inputs(
+        self.check_inputs(
```

Comment:
@kashif @dome272 let's try not to diverge from the design we have for other pipelines here. I changed this because a check function should not return anything - it should just check if the inputs are correct

Reply:
ok i'll move the modification logic out of these methods